src/server/views/dashboard/lists-edit.njk

```diff
@@ -57,105 +56,84 @@
+            Which country is this list for?
           </label>
         </h3>
-        {% if error.field === 'administrators' %}
+        {% if error.field === 'country' %}
           <span class="govuk-error-message">
             <span class="govuk-visually-hidden">Error:</span> {{ error.text }}
           </span>
         {% endif %}
-        <div id="administrators-hint-2" class="govuk-hint">
-          Enter the user’s email address, please separate multiple addresses with commas
-        </div>
-        <input
-          class="govuk-input"
-          id="administrators"
-          name="administrators"
-          type="text"
-          aria-describedby="administrators-hint administrators-hint-2"
-          value="{{list.jsonData.administrators or req.user.userData.email}}"
-        >
-      </fieldset>
-    </div>
-
 
-    <div class="govuk-form-group {{ "govuk-form-group--error" if error.field === 'validators' }}">
-      <fieldset class="govuk-fieldset">
-        <h3 class="govuk-label-wrapper">
-          <label class="govuk-label govuk-label--m" for="validators">
-            Users that can validate content
-          </label>
-        </h3>
-        {% if error.field === 'validators' %}
-          <span class="govuk-error-message">
-            <span class="govuk-visually-hidden">Error:</span> {{ error.text }}
-          </span>
+        {% if listId !== 'new' %}
+          <input class="govuk-input" name="country" type="text" value="{{list.country.name}}" disabled>
+        {% else %}
+          <select id="country-autocomplete" name="country" className="govuk-input--error">
+            <option value=""></option>
+            {% for country in countriesList %}
+              <option value="{{ country.value }}" {{-" selected='selected'" if list.country.name === country.value}}>{{ country.text }}</option>
+            {% endfor %}
+          </select>
         {% endif %}
-        <div id="validators-hint" class="govuk-hint">
-          Enter the user’s email address, please separate multiple addresses with commas
-        </div>
-        <input class="govuk-input" id="validators" name="validators" type="text" aria-describedby="validators-hint" value="{{list.jsonData.validators}}">
-      </fieldset>
-    </div>
+      </div>
 
+      <hr />
 
-    <div class="govuk-form-group {{ "govuk-form-group--error" if error.field === 'publishers' }}">
-      <fieldset class="govuk-fieldset">
-        <h3 class="govuk-label-wrapper">
-          <label class="govuk-label govuk-label--m" for="publishers">
-            Users that can publish content
-          </label>
-        </h3>
-        {% if error.field === 'publishers' %}
-          <span class="govuk-error-message">
-            <span class="govuk-visually-hidden">Error:</span> {{ error.text }}
-          </span>
+      <div class="govuk-form-group {{ "govuk-form-group--error" if error.field === 'publisher' }}">
```

Comment:
Per the trello card (https://trello.com/c/ZbJBJmdL/1490-annual-review-list-settings-annual-review-start-date#comment-6351a979702f6600e74e9b8e) this page also needs a back link at the top and bottom of the page, similar to the list item details page.